src/devices/Arduino/tests/xunit.runner.json

```diff
@@ -0,0 +1,4 @@
+{
+    "parallelizeAssembly": false,
+    "parallelizeTestCollections": false
+}
```

Comment:
this shouldn't be part of the folder. You can achieve the same adding a custom attribute to one of the files in the test project

Reply:
I was unable to get that to work as expected. It was always at least trying to run two test classes in parallel, which is also not working here.